src/main/java/ru/urururu/sanity/cpp/TypeParser.java

```diff
@@ -83,8 +83,8 @@ public void onModuleStarted(SWIGTYPE_p_LLVMOpaqueModule module) {
             for (int i = 0; i < arguments; i++) {
                 // process compile units
                 SWIGTYPE_p_LLVMOpaqueValue compileUnitMD = bitreader.getValue(values, i);
-                if (LlvmUtils.checkTag(compileUnitMD, DW_TAG_compile_unit)) {
-                    visit(compileUnitMD);
+                if (LlvmUtils.checkTag(compileUnitMD, null, DW_TAG_compile_unit)) {
+                    //visit(compileUnitMD);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)